python-avd/pyavd/_eos_designs/schema/schema_fragments/management_eapi.schema.yml

```
@@ -25,3 +25,36 @@ keys:
         default: true
       default_services:
         type: bool
+      vrfs:
```

to be non breaking, need default in the schema to add the mgmt interface VRF


```suggestion
      vrfs:
        default:
          - name: use_mgmt_interface_vrf
            enabled: true
```

For 6.0 default should use  `use_default_mgmt_method_vrf` (which is more correct but breaking) - please add a comment

---

Done!